openmc/mgxs/library.py

```diff
@@ -822,11 +822,12 @@ def load_from_file(filename='mgxs', directory='mgxs'):
         return pickle.load(open(full_filename, 'rb'))
 
     def get_xsdata(self, domain, xsdata_name, nuclide='total', xs_type='macro',
-                   order=None, tabular_legendre=None, tabular_points=33,
-                   subdomain=None):
+                   order=None, subdomain=None):
         """Generates an openmc.XSdata object describing a multi-group cross section
```

Comment:
Shorten the initial docstring description to no more than 2 lines so that it can be rendered inline with use of "CTRL+Tab" in IPython Notebook or other rich IDEs. A more detailed docstring can then be given in a separate paragraph below. This is how all of our other docstrings are formatted at least, for the above reason.